### PR TITLE
Add EIP-684 to Byzantium Meta

### LINKED ISSUE #2220: EIP-684 is missing from the repository
I'm going to refer to [this](https://github.com/ethereum/EIPs/issues/684) issue as EIP-684.

I've noticed that both Geth and Parity by default enable EIP-684, which is a consensus breaking change by default. EIP-684 was opened as an issue by Vitalik and promptly used as a patch 2 years ago, with no accompanying EIP.

Since this is in-fact a consensus breaking issue, we should have it documented as a valid EIP.

### PR DESCRIPTION
**Pull request by soc1c**
fix #2220 

to my knowledge, all clients implement it

### CODE REVIEW

File: EIPS/eip-609.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This specifies the changes included in the hard fork named Byzantium.
   - [EIP 214](https://eips.ethereum.org/EIPS/eip-214) (New opcode STATICCALL)
   - [EIP 649](https://eips.ethereum.org/EIPS/eip-649) (Difficulty Bomb Delay and Block Reward Reduction)
   - [EIP 658](https://eips.ethereum.org/EIPS/eip-658) (Embedding transaction status code in receipts)
+  - [EIP 684](https://eips.ethereum.org/EIPS/eip-684) (Prevent overwriting contracts)
```
Comment (axic) [inline EIPS/eip-609.md]: This is a broken url.

Comment (soc1c) [reply EIPS/eip-609.md]: yeah, right, 684 is not even an EIP yet

**Comment by axic** [comment]
There is a discussion in https://ethereum-magicians.org/t/eip-689-address-collision-of-contract-address-causes-exceptional-halt/3542 about this. I think we should do what was suggested in that discussion and then merge this.

I'd be against merging this PR as-is because currently all the "hard fork metas" are somewhat of a higher quality with linking to well-formatted EIPs. Compared to that this PR would link to a non-EIP-formatted link.

**Review by axic — CHANGES_REQUESTED** [review]
I think we should make 684 into an EIP and not linking to an issue here.

**Pull request closed without merging**